### PR TITLE
Readability + typo fixes to section 1

### PR DESCRIPTION
**Pull request by jonathancross**
I did my best to keep these changes as light as possible and avoid the temptation to reword things.  I hope you find it easier to review  :-)

### CODE REVIEW

**Comment by jonathancross** [comment]
Friendly ping @serhack @SamsungGalaxyPlayer

**Review by serhack — APPROVED** [review]
Thank you! It's really appreciated

File: chapters/1.md [hunk 1]
```diff
@@ -84,50 +84,50 @@ A few of the blockchain benefits are immediately apparent:
...
 
-* **Pseudo-privacy**: Unlike the banks, cryptocurrency ledgers never record real names such as “Maria” and “George” with the accounts. In fact, personal information is never necessary for generating a cryptocurrency wallet. Instead, George will access the funds pseudonymously, using his key for the 1eK5FSywkp address to which Maria broadcasted the money (from her account, 1BuUygisXY).
+* **Pseudo-privacy**: Unlike the banks, cryptocurrency ledgers never record real names such as “Maria” and “George” with the accounts. In fact, personal information is never necessary for generating a cryptocurrency wallet. Instead, George will access the funds pseudonymously, using his key for the 1eK5FSywkp address to which Maria broadcast the money (from her account, 1BuUygisXY).
 
-* **Reduced transaction costs* Trading with banks often is a great expense for fees; the cryptocurrencies does not need any middleman meaning less costs to send a transaction.
+* **Reduced transaction costs* Banks often charge high fees; cryptocurrencies do not require a middleman meaning less costs to send a transaction.
```
[the remark this answers is not in the document]
Comment (jonathancross) [reply chapters/1.md:89]: I find the word "circuits" confusing here. There might also be trademark issues with using their brand names as well.

How about this: _"Banks, credit cards and other payment systems"_ ?

[the remark this answers is not in the document]
Comment (serhack) [reply chapters/1.md:89]: I do agree. "Circuits" word does not sound so good.

[the remark this answers is not in the document]
Comment (jonathancross) [reply chapters/1.md]: Cool, done.

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Pull request merged**